src/tp/tp_manager.cpp

```
@@ -79,7 +79,7 @@ TpResult TpManager::segment_message(const Message& message, uint32_t& transfer_i
     TpTransfer transfer(transfer_id, message_id);
 
     // Segment the message
-    std::vector<TpSegment> segments;
+    platform::Vector<TpSegment> segments;
```

Cppcheck (2.21.0)

[style] The function 'set_request_id' is never used.

(unusedFunction)